lib/stdlib/doc/src/stdlib_app.xml

```diff
@@ -71,6 +71,19 @@
       <item>
         <p>Can be used to override the default keymap configuration for the shell.</p>
       </item>
+      <tag><marker id="shell_format_command"/><c>shell_format_command = {Mod, Func} | string() | default</c></tag>
+      <item>
```

Comment:
Add a cross link to `shell:shell_format_func/1` somewhere in these docs.

And also, should this config variable not be named the same as the function?